x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/edit_exception_flyout/index.tsx

```diff
@@ -367,6 +388,24 @@ const EditExceptionFlyoutComponent: React.FC<EditExceptionFlyoutProps> = ({
     prefix: 'exceptionFlyoutTitle',
   });
 
+  const confirmModal = useMemo(() => {
+    const { title, body, confirmButton, cancelButton } = CONFIRM_WARNING_MODAL_LABELS(
+      listType === ExceptionListTypeEnum.ENDPOINT ? ENDPOINT_EXCEPTION : RULE_EXCEPTION
```

Comment:
So, I can see this PR affects not only endpoint exceptions, but rule exceptions too.

We already had help text, that points wildcard is used with a wrong operator.

<img width="2545" alt="Screenshot 2024-05-17 at 12 20 35" src="https://github.com/elastic/kibana/assets/92328789/a84e62c5-b981-4dd4-bad4-f5a331eb06d7">

Now, we would have additional waring and confirm modal. Three different UI elements highlighting the same possible misusing of wildcard.

cc: @ARWNightingale , @approksiu - are we good with such behaviour for rule exceptions?

Reply:
The additional warning callout and confirm modal were requests from product to make sure users were addressing the issue in the value. We had several sdhs where users were having trouble diagnosing why their entries were not effective and the smaller warning underneath the value is easy to miss. 

@caitlinbetz  did you have more to add on the design decision?

Reply:
We should think about how to de-duplicate warnings. I thnk confirmation popup is good, but we should keep only one of the warnings - either under the value or under all conditions. cc @ARWNightingale @caitlinbetz

Reply:
Thanks @approksiu . We currently have two because 1. we wanted the user to be aware of the potential issue on the form, as it was being created, and 2. wanted the user to confirm the exception if we notice a potential configuration error. So - I think they do both currently serve a purpose. 

I think the "third" message, under the value, we can remove (as long as I'm not missing anything @parkiino.) I believe it was added to maintain consistency across some other forms, but again @parkiino let me know your thoughts on removing this one.

Reply:
@caitlinbetz  i think we probably want to keep the one under the value since they show the error per individual condition if there are multiple stringed together with the "AND" or "OR". It seems like @approksiu was suggesting we take out the single callout under all the conditions, but then that would make the design inconsistent with trusted apps and event filters. I'm not sure if it makes sense to only have it for endpoint exceptions and not rule exceptions too, if we tried to split it up that way.

Reply:
@parkiino after discussing with @approksiu  - we are okay to leave this functionality as is for the time being. Thank you.